packages/backend-common/src/database/DatabaseManager.ts

```diff
@@ -136,13 +145,18 @@ export class DatabaseManager {
     );
   }
 
+  private getUsePluginSchemasConfig(): boolean {
```

Comment:
Perhaps call this `shouldUsePluginSchemas`?

Reply:
Updated to `getPluginDivisionModeConfig` following the next comment!